src/Features/Core/Portable/Completion/Providers/AbstractInternalsVisibleToCompletionProvider.cs

```diff
@@ -243,6 +248,32 @@ public override async Task<CompletionChange> GetChangeAsync(Document document, C
             return CompletionChange.Create(textChange);
         }
 
+        private static async Task<TextSpan> GetTextChangeSpanAsync(Document document, TextSpan startSpan, CancellationToken cancellationToken)
+        {
+            var result = startSpan;
+            var syntaxFactsService = document.GetLanguageService<ISyntaxFactsService>();
+            var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var token = root.FindToken(result.Start);
+            if (syntaxFactsService.IsStringLiteral(token) || syntaxFactsService.IsVerbatimStringLiteral(token))
+            {
+                result = token.Span;
+                var contentStartsAt = token.Text.IndexOf('"');
+                contentStartsAt = contentStartsAt >= 0
+                    ? contentStartsAt + 1
+                    : contentStartsAt;
```

Comment:
nit.  if contentStartsAt was less than 0, then i wouldn't do any work.

Reply:
I check if the token is a (Verbatim)StringLiteral before. I think IndexOf will never return -1. Should I simplify that to
`var contentStartsAt = token.Text.IndexOf('"')+1;`?

Reply:
> I think IndexOf will never return -1. 

I don't trust things that much.  I would assume the worst and have the code check for that.

Note: i don't think you need any of this if we use the GetWordSpan approach suggested above.  That would be my preference on how we code up this solution.